<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>806</width>
      <height>626</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Wybór parametrów trasy</string>
     </attribute>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>371</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Wygeneruj graf</string>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>341</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Mapa odległości</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Wysokość:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="heightBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Szerokość</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="widthBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>9</number>
        </property>
       </widget>
       <widget class="QSlider" name="heightSlider">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="widthSlider">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>60</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>341</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Mapa wysokości</string>
       </property>
       <widget class="QComboBox" name="heightProfileBox">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Losowo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Góra w centrum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dolina w centrum</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Wybór profilu</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>341</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Krawędzie</string>
       </property>
       <widget class="QCheckBox" name="poziomoBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Poziome</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="pionowoBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pionowe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="skos1Box">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Skosy w prawo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="skos2Box">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Skosy w lewo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSlider" name="percentageSlider">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>20</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Zróżnicowanie długości krawędzi:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="percentageBox">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string extracomment="%"/>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="cubeBox">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Czy kwadrat?</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="createGraphButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>470</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Wygeneruj graf</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>341</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Betonowość</string>
       </property>
       <widget class="QComboBox" name="betonBox">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Pion+poziom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Losowo</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Wybór profilu</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>341</width>
         <height>71</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="title">
        <string>Zapis do pliku</string>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nazwa pliku:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="filenameOutEdit">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>test</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Wczytaj graf z pliku</string>
      </property>
      <widget class="QPushButton" name="drawFromFileButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rysuj graf z pliku</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="filenameInEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>pelny10x10</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Wizualizacja grafu</string>
     </attribute>
     <widget class="QWidget" name="graphWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClose">
    <property name="title">
     <string>Zamknij</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTesty">
    <property name="title">
     <string>Testy</string>
    </property>
   </widget>
   <addaction name="menuClose"/>
   <addaction name="menuTesty"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>drawFromFileButton</tabstop>
  <tabstop>heightSlider</tabstop>
  <tabstop>heightBox</tabstop>
  <tabstop>widthBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>widthSlider</tabstop>
  <tabstop>heightProfileBox</tabstop>
  <tabstop>poziomoBox</tabstop>
  <tabstop>pionowoBox</tabstop>
  <tabstop>skos1Box</tabstop>
  <tabstop>skos2Box</tabstop>
  <tabstop>percentageSlider</tabstop>
  <tabstop>percentageBox</tabstop>
  <tabstop>createGraphButton</tabstop>
  <tabstop>betonBox</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>850</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
